src/js/owl.video.js

```diff
@@ -219,7 +219,7 @@
 		} else if (video.type === 'vzaar') {
 			$.ajax({
 				type: 'GET',
-				url: '//vzaar.com/api/videos/' + video.id + '.json',
+				url: 'http://vzaar.com/api/videos/' + video.id + '.json',
```

Comment:
Shouldn't this be also https://? As this might throw a warning on SSL secured sites.